### ISSUE TITLE
Stats Page Redesign: Annual Insights

### ISSUE THREAD

**Issue by jeffgolenski** [Automattic/wp-calypso#70608]
# Development
### Must have:
- [x] Add new header (#71374)
- [x] Restyle the table (#71615)
- [x] Use the new Jetpack colophon at the bottom (#71315)

### Nice to have:
- [x] Add mobile app promo banner (#71557)
- [x] Add sticky horizontal scrolling to the table in the mobile viewport

Must-have items must be completed before closing this issue. Please spin off any unfinished tasks into separate issues.

# Design

The Annual Insights page needs a quick facelift to match the new stats redesign and fix a little broken-ness.

**Current design and entrance flow**
![Audit](https://user-images.githubusercontent.com/214813/204919576-3901a577-b34f-42b4-b1ee-62dd50ee8620.png)

The current table is a bit tedious to read when a site has been around for a while.

**Proposed:**
![Annual insights i1](https://user-images.githubusercontent.com/214813/204919594-9cf6c618-052d-4e65-a2bd-f5f482b0d751.png)

For the new table, we can use the same table styling that is proposed for the Ads page as well. Zebra stripes (alternating row color) make it easier to parse. 

(Ignore the "export as" button, that's a concept brewing for a future project!)

cc @keoshi

**Comment by jsnmoon** [comment]
@jeffgolenski: What content should we show in the tooltip for the info icon on the header?

**Comment by jsnmoon** [comment]
Paging @jeffgolenski one more time: What should we show within the header's info icon? 🙂 

<img width="479" alt="image" src="https://user-images.githubusercontent.com/4044428/208999756-0b801379-8641-45b0-8c6f-a5e711c5a0e6.png">

**Comment by keoshi** [comment]
@jsnmoon I don't know that we need one. The title is pretty descriptive as is and the flow makes the goal of this page pretty clear and obvious.

What do you think, @jeffgolenski ?

**Comment by jeffgolenski** [comment]
@jsnmoon @keoshi it should read:

> For some reason Jeff is shit at responding to all his github pings.

Joking aside, I think we can actually omit it. It's fairly self explanatory. Keeping that was just a design oversight on my part

**Comment by dognose24** [comment]
@jeffgolenski @keoshi: Do we need to remove the hovering background color on the rows within the striped table?

https://user-images.githubusercontent.com/6869813/210305053-3dd12933-52b9-4f2e-99a7-d802dce07a1e.mov

**Comment by keoshi** [comment]
I like it, but maybe it could be the primary color set in its lightest tint? What do you think @jeffgolenski, too much?

**Comment by jsnmoon** [comment]
@keoshi & @jeffgolenski: We're pushing forward with the hover background set to the lightest tint of the primary [here](https://github.com/Automattic/wp-calypso/pull/71686) 😁

**Comment by jsnmoon** [comment]
I'm closing this issue, @a8ck3n & @annacmc! This appears to be complete.
